### PR TITLE
FR text updates for settings page

### PR DESCRIPTION
**Pull request by timarney**
# Summary | Résumé

Updates translation strings for settings page

Screen Shot 2023-03-28 at 1 33 44 PM

Screen Shot 2023-03-28 at 1 35 08 PM

# Test instructions | Instructions pour tester la modification

Visit setting page as a logged in / logged out user
```
fr/form-builder/settings
```

# Pull Request Checklist

Please complete the following items in the checklist before you request a review:

- [x] Have you completely tested the functionality of change introduced in this PR? Is the PR solving the problem it's meant to solve within the scope of the related issue?
- [x] The PR does not introduce any new issues such as failed tests, console warnings or new bugs.
- [ ] If this PR adds a package have you ensured its licensed correctly and does not add additional security issues?
- [ ] Is the code clean, readable and maintainable? Is it easy to understand and comprehend.
- [ ] Does your code have adequate comprehensible comments? Do new functions have [docstrings](https://tsdoc.org/)?
- [ ] Have you modified the change log and updated any relevant documentation?
- [ ] Is there adequate test coverage? Both unit tests and end-to-end tests where applicable?
- [ ] If your PR is touching any UI is it accessible? Have you tested it with a screen reader? Have you tested it with automated testing tools such as axe?

### CODE REVIEW

**Comment by timarney** [comment]
@anikbrazeau 

I started this PR to get the settings screen updated

There's a couple of error messages that need translation 

https://github.com/cds-snc/platform-forms-client/pull/1791/files#diff-a6e486b021382fa7935492752ec9890787007f55c22fdc5c300ec95e11355a7bR456

--

The EN + FR for subject line doesn't seem to line up between english and french so those also need to be fixed in this PR.

```
English subject line 
- When form responses are submitted in English. 

French subject line
 - When form responses are submitted in French.

File: public/static/locales/fr/form-builder.json [hunk 1]
```diff
@@ -440,29 +440,29 @@
...
-      "text1": "[FR] Submitted responses will appear on the",
-      "text2": "[FR] Responses page"
+      "text1": " Les réponses reçues apparaîtront à la page Réponses",
+      "text2": "la page Réponses"
     },
-    "emailOption": "[FR] Email the responses to an inbox",
-    "saveButton": "[FR] Save changes",
+    "emailOption": "Les réponses soumises seront envoyées par courriel à cette boîte de réception.",
+    "saveButton": "Enregistrer les modifications",
     "savedSuccessMessage": "[FR] Your changes have been saved.",
```
Comment (anikbrazeau) [inline public/static/locales/fr/form-builder.json:456]:
```suggestion
    "savedSuccessMessage": "Vos modifications ont été enregistrées.",
```

**Comment by anikbrazeau** [comment]
@timarney thanks for flagging! here are the corresponding translations, let me know if anything else is missing.

EN: `English subject line`
FR: `Ligne d'objet en anglais`

EN: `When form responses are submitted in English.`
FR: `Lorsque les réponses au formulaire sont soumises en anglais.`

EN: `French subject line`
FR: `Ligne d'objet en français`

EN: `When form responses are submitted in French. `
FR: `Lorsque les réponses au formulaire sont soumises en français.`

**Comment by timarney** [comment]
> @anikbrazeau
> 
> I started this PR to get the settings screen updated
> 
> There's a couple of error messages that need translation
> 
> https://github.com/cds-snc/platform-forms-client/pull/1791/files#diff-a6e486b021382fa7935492752ec9890787007f55c22fdc5c300ec95e11355a7bR456
> 
> --
> 
> The EN + FR for subject line doesn't seem to line up between english and french so those also need to be fixed in this PR.
> 
> ```
> English subject line 
> - When form responses are submitted in English. 
> 
> French subject line
>  - When form responses are submitted in French. 
> ```

Great thanks I pushed some updates

**Review by dsamojlenko — APPROVED** [review]

**Pull request merged**